tests/methods/ParseTest.php

```diff
@@ -198,4 +198,41 @@ public function testAutoQuotes($file, $enclosure) {
         $this->assertArrayHasKey('column1', $csv->data[0], 'Data parsed incorrectly with enclosure ' . $enclosure);
         $this->assertEquals('value1', $csv->data[0]['column1'], 'Data parsed incorrectly with enclosure ' . $enclosure);
     }
+
+    public function countRowsProvider(){
```

Comment:
Theses functions are fine. Since technically the CSV is not parsed, please put them into a new test file.

Reply:
Done